src/data_manager/collectors/scrapers/integrations/git_scraper.py

```diff
@@ -80,20 +84,25 @@ def _harvest_repository(
             return []
 
         resources: List[ScrapedResource] = []
+        repo = Repo(repo_path)
         for markdown_path in docs_dir.rglob("*.md"):
             current_url = base_site_url + \
                 markdown_path.relative_to(docs_dir).with_suffix("").as_posix()
 
             logger.info(f"Indexing Git doc: {current_url}")
             text_content = markdown_path.read_text(encoding="utf-8")
+            created_at = self._get_creation_date(repo=repo, file_path=markdown_path)
+            last_updated_at = self._get_last_updated_date(repo=repo, file_path=markdown_path)
             resource = ScrapedResource(
                 url=current_url,
                 content=text_content,
```

Comment:
We could think about putting the file name, date, repo, etc. in the text_content to make it easier for things like BM25 to find files via query. What do you think?